src/org/labkey/test/components/CustomizeView.java

```diff
@@ -194,12 +194,20 @@ public void saveCustomView(String name, boolean shared, boolean inherit)
         {
             saveWindow.shareCheckbox.check();
         }
+        else
+        {
+            saveWindow.shareCheckbox.uncheck();
+        }
```

Comment:
I'm worried that this will interfere with tests that depend on the current behavior (where `false` actually means to leave the default).
I'd suggest changing these flags to be nullable and update the other `saveCustomView` overloads to pass in `null` instead of false.
Then set the checkbox only if the flag isn't null.
Something like this:
```
public void saveCustomView(@Nullable String name) { saveCustomView(name, null, null); }

public void saveCustomView(String name, @Nullable Boolean shared) { saveCustomView(name, shared, null); }

public void saveCustomView(String name, @Nullable Boolean shared, @Nullable Boolean inherit)
{
    SaveWindow saveWindow = clickSave();

    if (shared != null)
    {
        saveWindow.shareCheckbox.set(shared);
    }

    if (inherit != null)
    {
        saveWindow.inheritCheckbox.set(inherit);
    }

etc.
```

Reply:
Sounds reasonable. I updated the overloads and other tests that were currently passing in false for either of those fields.